Simonyi/Non-Standard-Operations/MTCS/MTHexRot/MTHex-PXI-Controller-Reboot.rst

```diff
@@ -213,16 +213,17 @@ Hard Reboot
 
     .. figure:: /Simonyi/Non-Standard-Operations/_static/mtrot-controller-pxi-reboot-1.jpeg
```

Comment:
You can remove this figure becuase - 
1. outdated
2. you can directly link the PDU outlet description page

Reply:
Why is this figure outdated? The PDU for MTRot and MTCamHexapod is still the same. 
Once we release the PDU pages we can link from here.

Reply:
I will leave this figure as I believe this PDU hasn´t changed.